stable/gcpdisk-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: 0.6.0
+description: Google Compute Engine Persistent Disk (GCE PD) Container Storage Interface (CSI) Storage Plugin
+name: gcpdisk-csi-driver
+maintainers:
+  - name: sebbrandt87
+version: 0.6.0
+kubeVersion: ">=1.16.0"
```

Comment:
cannot it be used with 1.15 🤔  ? we didn't face this restriction for our existing addons :/

Reply:
we did for azure as well, pointed it to 0.15.0
we should not have it backward compatible out of my perspective, as we implement it right now and not in the past.

Reply:
we didn't https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/azuredisk-csi-driver.yaml#L15

Reply:
Konvoy 1.3 supports 1.15 and 1.16(default), if this addon is part of 1.16 we need to leverage a logic to avoid deploying it on 1.15 cluster. At this moment, this logic is not in place, so that will be a blocker for this addon. IF you confirm it only works on 1.16, if it can work on 1.15 I don't see why to set this value here.

Reply:
I'd also be interested on see what it is blocking this chart to run on 1.15 🤔

Reply:
nothing blocks it, changed it